sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_shared/utils.py

```diff
@@ -9,7 +9,7 @@
 import calendar
 from typing import cast, Tuple, Optional
 from datetime import datetime
-from azure.core.serialization import TZ_UTC
+from msrest.serialization import TZ_UTC
```

Comment:
This change needs to be reverted - can you update this in all of the _shared files?
None of the SDKs should be using msrest directly.